packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -163,6 +163,11 @@ export class Viewer {
    */
   @Event() public tokenExpired!: EventEmitter<void>;
 
+  /**
+   * Emits an event when the jwt for a session gets updated
+   */
+  @Event() public jwtRefresh!: EventEmitter<string>;
```

Comment:
How about we instead have a more generic "connected" event with an event payload that we can add additional info to later? I could see developers wanting to know when the SDK is in different connections states, e.g. `connecting | connected | connection_failed | disconnected`